OPAL/tac/src/main/scala/org/opalj/tac/fpcf/analyses/cg/AndroidICCAnalysis.scala

```diff
@@ -307,155 +323,165 @@ class AndroidICCAnalysis(val project: SomeProject) extends FPCFAnalysis {
     /**
      * Searches all Intents and callbacks in the project.
      */
-    def searchIntentsAndCallbacks(): Unit = {
-        val source = Source.fromInputStream(this.getClass.getResourceAsStream("/org/opalj/tac/fpcf/analyses/cg/AndroidCallbackList.txt"))
-        val callbackList = source.getLines.toList
-        source.close
+    def searchIntentsAndCallbacks(): ProperPropertyComputationResult = {
         val dec = project.get(DeclaredMethodsKey)
-        val leftOverMethods = mutable.Queue[Method]()
+        val partialResults = ListBuffer.empty[ProperPropertyComputationResult]
         project.allMethodsWithBody.filter(dec(_) != null).foreach(m ⇒ {
             val tacaiEP = propertyStore(m, TACAI.key)
             if (tacaiEP.hasUBP && tacaiEP.ub.tac.isDefined) {
-                processMethod(m, callbackList)
-            } else leftOverMethods += m
-        })
-        while (leftOverMethods.size > 1) {
-            val m = leftOverMethods.dequeue()
-            val tacaiEP = propertyStore(m, TACAI.key)
-            if (tacaiEP.hasUBP && tacaiEP.ub.tac.isDefined) {
-                processMethod(m, callbackList)
+                processMethod(m)
             } else {
                 leftOverMethods += m
+                partialResults += InterimPartialResult(Nil, Set(tacaiEP), continuationForTAC(m))
             }
+        })
+        if (leftOverMethods.isEmpty) {
+            startSecondPhase(project)
+        } else Results(partialResults)
+    }
+
+    private[this] def continuationForTAC(m: Method)(eps: SomeEPS): PropertyComputationResult = {
```

Comment:
I'm confused. Aren't continuations required to return a `ProperPropertyComputationResult`, i.e., not `NoResult`?

Reply:
It did work with NoResult, but it might cause trouble I am not aware of. At this point there is no result available. Is there any way I can return a ProperPropertyComputationResult, that acts like NoResult?

Reply:
It seems I was wrong and InterimPartialResults may actually return NoResult. So it should not be a problem.